imageprocessor/views.py

```diff
@@ -139,3 +144,31 @@ def register(request):
         return HttpResponseRedirect(reverse('login'))
     context['form'] = form
     return render(request, 'register.html', context)
+
+class ClassifyAPI(APIView):
```

Comment:
Can you update this function to support uploading multiple image files?  Right now, this runs detection on the first file and ignores the rest. I would look at the updated classify function to see how to do this

Reply:
I dont think I can get to that change today (this sprint) so I'll add a story to backlog.

Reply:
https://trello.com/c/VRgmdUnV/90-1-as-a-3rd-party-developer-i-want-to-submit-multiple-images-in-a-single-request-to-the-classify-api

Reply:
fair enough... It the future, we should consolidate the code so that the API is doing the same thing as the view. The only differences should be whether we're adding data to the context or returning data in the form of JSON